packages/responsive/src/useMediaQueries.tsx

```diff
@@ -0,0 +1,168 @@
+import React, { FC, useEffect, useState } from "react"
+import { useTheme } from "@kaizen/design-tokens"
+
+type Props = { [key: string]: string }
+
+const minus1Px = (breakpoint: string) => `${parseInt(breakpoint, 10) - 1}px`
+
+export const useMediaQueries = (
+  propQueries: Props
```

Comment:
I reckon @OliBridgman’s previous note to default to an empty object is still a good one. No value in forcing consumers to pass through empty objects in the caller all the time.

Reply:
Yeah I want to do it but ran into problems with TS given the dynamic nature of props coming in. i.e. I wasn't able to make `[key: string]: string` optional, but I can have another go at it. Let me know if you know of a good way to do it.

Reply:
Ah, right, I’ll have a squiz and see if I can see the issue.

Reply:
You shouldn’t need to make the arguments optional, it should be enough to set a default:

```ts
export const useMediaQueries = (
  propQueries: Props = {}
): {
```

That’ll allow the caller to not pass the default `useMediaQueries()` without making anything "optional" in the arguments themselves.

Reply:
Nice, ty!